### DISCUSSION TITLE
Image field validation

### DISCUSSION THREAD

**Discussion by gepopp** [laravel/nova-issues#4100] [Q&A] [answered]
- Laravel Version: 9.1
- Nova Version: 4.2
- PHP Version: 8.1

### Description:

```
use Laravel\Nova\Fields\Image as ImageField;

...

 ImageField::make( 'Bild', 'photo_path' )
                      ->disk( 's3' )
                      ->storeAs( function ( Request $request ) {
                          return $request->photo_path->getClientOriginalName();
                      } )->rules( [ 'required' ] ),

```

On the resource edit screen, if an image is allready set, the "redquired" rule forces to upload a new image:
![image](https://user-images.githubusercontent.com/2655976/165034061-7ccc8de3-0f54-4024-9921-f675dfa6d9a9.png)

**Comment by arzola** [comment]
I can confirm this is happening in the latest Nova version

**Comment by davidhemphill** [comment, marked as answer]
You need to use the `updateRules` method on your field definition: https://nova.laravel.com/docs/4.0/resources/validation.html#update-rules

**Reply by gepopp** [reply]
And which rule should i user there? ich i user the "nullable" rule the image can be deleted and it can be saved without an image? But that not what it should do, i just want to ba able to update all other fields withput setting a new picture if one is alaredy present?
